torch/nn/_functions/thnn/rnnFusedPointwise.py

```diff
@@ -0,0 +1,59 @@
+import torch
+from torch.autograd.function import Function, InplaceFunction
+from torch._thnn import type2backend
+
+
+class GRUFused(Function):
+    def forward(self, input_gate, hidden_gate, hx, ibias=None, hbias=None):
+        hy = input_gate.new()
+        type2backend[type(input_gate)].GRUFused_updateOutput(
```

Comment:
@apaszke I wonder if this could be the reason I was seeing some extra overhead on-top of autograd. Will make the change, and will investigate overhead. Good catch, thanks.